__tests__/reactTags.test.js

```diff
@@ -228,6 +228,29 @@ describe('Test ReactTags', () => {
 
       expect(tags).to.deep.have.same.members(expected);
     });
+
+    test('should allow pasting text only up to maxLength characters', () => {
+      const tags = [];
+      const $el = mount(mockItem({ handleAddition(tag) {
+            tags.push(tag);
+          }, maxLength: 5 }));
+
+      const $input = $el.find('.ReactTags__tagInputField');
+
+      $input.simulate('paste', {
+        clipboardData: {
+          getData: () => 'Thimbleberry',
```

Comment:
can you assign `Thimbleberry` to a variable and use that variable here a nd in line 247 ?

Reply:
@imtanmoy seems like this got missed out